packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -850,7 +850,7 @@ Or read our docs at https://www.prisma.io/docs/concepts/components/prisma-client
           attributes: {
             method: params.action,
             model: params.model,
-            name: `${params.model}.${params.action}`,
```

Comment:
So for example, `$queryRaw`?

Reply:
Oh,  right, completely forgot that I did this part. 
Yes, this is a fix for raw methods.